detox/index.d.ts

```diff
@@ -785,8 +866,8 @@ declare global {
              */
             delete?: boolean;
             /**
-             * Detox can start the app with additional launch arguments
-             * The added launchArgs will be passed through the launch command to the device and be accessible via [[NSProcessInfo processInfo] arguments]
+             * Arguments to pass-through into the app.
+             * Refer to the [dedicated guide](https://github.com/wix/Detox/blob/master/docs/APIRef.LaunchArgs.md) for complete details.
              */
             launchArgs?: any;
```

Comment:
@d4vidi, don't you find this weird? I don't remember we ever had such an API... 😮 🤔

Reply:
hmmm wdym?

Reply:
Check the actual implementation of `Device.js` and find it.

Reply:
Oh, Github you're drunk. :( That was just 306 / 362 line jump. No worries, resolving.